src/models/reposyncstate.ts

```diff
@@ -30,6 +31,7 @@ export class RepoSyncState extends Model {
 	repoCreatedAt?: Date;
 	syncUpdatedAt?: Date;
 	syncCompletedAt?: Date;
+	config: Config;
```

Comment:
Later, we might want to add a `config` field to the `Subscription` and `Installation` types as well to act as a fallback when there is no repo-scoped config.